### PR TITLE
DEV-2834 fixed meterElec reporting strategy

### PR DESCRIPTION
**Pull request by kacperdrobny**

### CODE REVIEW

File: src/internal/easee/thing.go [hunk 1]
```diff
@@ -66,9 +66,8 @@ func (t *thingFactory) Create(ad adapter.Adapter, publisher adapter.Publisher, t
 			Controller:    controller,
 		},
 		MeterElecConfig: &numericmeter.Config{
-			Specification:     t.meterElecSpecification(ad, thingState, groups),
-			Reporter:          controller,
-			ReportingStrategy: controller.ReportRequired(),
```
Comment (boguslaw-wojcik) [inline src/internal/easee/thing.go:71]: As discussed please add custom strategy with AtLeastEvery 1m

**Review by boguslaw-wojcik — CHANGES_REQUESTED** [review]
Strategy.

**Review by rafalmnich — APPROVED** [review]

File: src/internal/easee/controller.go [hunk 2]
```diff
@@ -151,7 +150,7 @@ func (c *controller) ChargepointCableLockReport() (*chargepoint.CableReport, err
 }
 
 func (c *controller) ChargepointCurrentSessionReport() (*chargepoint.SessionReport, error) {
-	if err := c.checkConnection(); err != nil {
+	if err := c.checkSignalRConnection(); err != nil {
```
Comment (rafalmnich) [inline src/internal/easee/controller.go:153]: Shouldn't we block any report, when the charger reports "offline"?

**Comment by kacperdrobny** [comment]
discarded PR continued in https://github.com/futurehomeno/edge-easee-adapter/pull/33

**Pull request merged**